redisinsight/api/src/modules/database/database.service.ts

```diff
@@ -273,4 +347,14 @@ export class DatabaseService {
       { groups: ['security'] },
     ));
   }
+
+  private checkSecurityFields(model: Database): Database {
```

Comment:
Let's name it "hide" securty fields or "convert" or whatever, but not "check"

Reply:
rename checkSecurityFields with hideSecurityFields